apps/aecore/test/aec_fork_block_settings_tests.erl

```diff
@@ -12,38 +13,70 @@
 
 genesis_accounts_test_() ->
     release_based(?ROOT_DIR ++ "/.genesis",
+                  undefined,
                   fun ?TEST_MODULE:genesis_accounts/0,
                   none,
                   none,
                   genesis_accounts_file_missing).
 
 minerva_accounts_test_() ->
     release_based(?ROOT_DIR ++ "/.minerva",
+                  undefined,
                   fun ?TEST_MODULE:minerva_accounts/0,
                   none,
                   none,
                   minerva_accounts_file_missing).
 
 fortuna_accounts_test_() ->
     release_based(?ROOT_DIR ++ "/.fortuna",
+                  undefined,
                   fun ?TEST_MODULE:fortuna_accounts/0,
                   none,
                   none,
                   fortuna_accounts_file_missing).
 
 lima_accounts_test_() ->
     release_based(?ROOT_DIR ++ "/.lima",
+                  undefined,
                   fun ?TEST_MODULE:lima_accounts/0,
                   fun ?TEST_MODULE:lima_contracts/0,
                   lima_contracts_file_missing,
                   lima_accounts_file_missing).
 
-release_based(Dir, ReadAccountsFun, ReadContractsFun, CMissingErr, AMissingErr) ->
+generic_accounts_test_() ->
```

Comment:
What the term "generic" means?

Reply:
It is meant to mean the opposite of hard coded. I could change it to configurable.